azure-kusto-ingest/src/resource_manager/utils.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::Result;
+use azure_kusto_data::models::TableV1;
+
+/// Helper to get a column index from a table
+// TODO: this could be moved upstream into Kusto Data - would likely result in a change to the API of this function to return an Option<usize>
```

Comment:
Probably a good idea, we need a better interface for tables in general

Reply:
I've changed the syntax so it would be ready to move into the data crate - I think the move itself can be done in a separate PR